x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_nodejs/ml/abnormal_trace_durations_nodejs.json

```diff
@@ -0,0 +1,40 @@
+{
+  "job_type": "anomaly_detector",
+  "groups": [
+    "apm"
+  ],
+  "description": "APM NodeJS: Identifies trace transactions that are processing more slowly than usual.",
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "high_mean(\"transaction.duration.us\") by \"transaction.name\"",
+        "function": "high_mean",
+        "field_name": "transaction.duration.us",
+        "by_field_name": "transaction.name"
+      }
+    ],
+    "influencers": [
+      "transaction.name",
+      "trace.id",
+      "service.name"
+    ]
+  },
+  "allow_lazy_open": true,
+  "analysis_limits": {
+    "model_memory_limit": "256mb"
+  },
+  "data_description": {
+    "time_field": "@timestamp"
+  },
+  "custom_settings": {
+    "created_by": "ml-module-apm-nosejs",
```

Comment:
~`"created_by": "ml-module-apm-nosejs"`~ -> `"created_by": "ml-module-apm-nodejs"`

Reply:
Fixed in https://github.com/elastic/kibana/pull/53792/commits/144c0ba1c62d4da9d1fc35a03d930a1537bfb205